task_sdk/src/airflow/sdk/execution_time/supervisor.py

```diff
@@ -374,9 +353,6 @@ def start(
             logs=read_logs,
         )
 
-        # Tell the task process what it needs to do!
-        proc._on_child_started(what, path, requests_fd)
```

Comment:
Calling this is now the responsibility of the subclasses overridden `start` methods